index.html

```diff
@@ -4,9 +4,29 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css"/>
+</script>
     <title>Document</title>
 </head>
 <body>
-    <h1>Text</h1>
+    <header>
+        <p class="my-logo">My Logo</p>
+    </header>
```

Comment:
- [ ] Please closely follow the figma design in every part of the project [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
This is your current design
![closefigma](https://user-images.githubusercontent.com/55596970/153298912-5d01d1e6-176d-4e7c-aca3-411bbbbb923b.png)
This is what is expected 
![expectedfigma](https://user-images.githubusercontent.com/55596970/153299022-9e375047-4c86-44e2-905a-8bfe4d545826.png)

Reply:
@CollinsTatang  Thank you, I made all the changes requested.